arangod/RestServer/Metrics.h

```diff
@@ -539,9 +547,10 @@ template<typename Scale> class Histogram : public Metric {
 
  private:
   Metrics::hist_type _c;
-  Scale _scale;
-  value_type _lowr, _highr;
-  size_t _n;
+  Scale const _scale;
+  size_t const _n;
+  std::atomic<value_type> _lowr;
+  std::atomic<value_type> _highr;
   std::atomic<value_type> _sum;
```

Comment:
I don't think this change is required. Lower and higher should newer be modified after creating a histogram. If we do have code that does that, it seem wrong.

Reply:
No, these track the highest/lowest value ever recorded. Since these were not updated atomically, the previous implementation might report incorrect values.

Reply:
These values are not required, so I changed the code to only track them in maintainer mode for debugging.